contrib/babelfishpg_tds/error_mapping.txt

```diff
@@ -179,4 +179,7 @@ XX000 ERRCODE_INTERNAL_ERROR	"The table-valued parameter \"%s\" must be declared
 0A000 ERRCODE_FEATURE_NOT_SUPPORTED	"Column name or number of supplied values does not match table definition."	SQL_ERROR_213	16
 42501 ERRCODE_INSUFFICIENT_PRIVILEGE	"Only members of the sysadmin role can execute this stored procedure."	SQL_ERROR_15003	16
 42809 ERRCODE_WRONG_OBJECT_TYPE	 "The target \"%s\" of the OUTPUT INTO clause cannot be a view or common table expression."	SQL_ERROR_330	16
-
+22008 ERRCODE_DATETIME_VALUE_OUT_OF_RANGE	"The datediff function resulted in an overflow. The number of dateparts separating two date/time instances is too large. Try to use datediff with a less precise datepart" SQL_ERROR_535 16
```

Comment:
have we validated the transactional/control behavior for new error mappings? we need to make sure we are compatible with TSQL for these. There are automated scripts to validate an error's behavior. Please talk to Dipesh for more details.

Reply:
Zhibai and I validated the transactional/control behavior for the new error mappings and it is compatible with TSQL

Reply:
Yes , I have asked in purpose and it has test cases regarding the transaction behavior